src/agents/definitions/log-analysis-summarizer.js

```
@@ -28,6 +29,7 @@ export default {
 2026-06-04T03:15:20Z INFO  [order-service] DB connection acquired successfully
 2026-06-04T03:15:21Z INFO  [order-service] Service recovering — 200 responses resuming`,
     log_source: "Application (Node.js / Python / Java / etc.)",
+    focus_area: ["Errors & Root Cause"],
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)